src/tests/test_dynamic_api/remote/test_views.py

```diff
@@ -800,32 +738,6 @@ def respond(request):
     assert data == expect
 
 
-@pytest.mark.django_db
-def test_hcbrk_listing_by_bsn(api_client, fetch_auth_token, router, hcbrk_dataset, urllib3_mocker):
```

Comment:
is deze test niet meer relevant?

Reply:
Nee, we laten nu gewoon alles door. Ik zal de gebruikers zeggen dat ze de BSN-optie maar even niet moeten gebruiken tot we alles met logging e.d. uitgezocht hebben.